RobsiZocktUni/Project-Exit
# Finished Clothing Puzzle

To Test:
- Pick up the clothing.
- Put the correct clothing on each mannequin (From left to right: Winter Outfit, Grieving Dress, Rain Outfit, Summer Dress).

Added prefab for all clothing with Doll_Clothing.cs.

Modified DollHousePuzzleManager.cs:
- Implemented a check function to verify if all mannequins have the correct clothing.

Modified Mannequin.cs:
- Calls the DollHousePuzzleManager check function whenever clothing is added.
